llama-index-core/llama_index/core/workflow/workflow.py

```diff
@@ -299,6 +322,8 @@ def is_done(self) -> bool:
     async def _done(self, ctx: Context, ev: StopEvent) -> None:
         """Tears down the whole workflow and stop execution."""
         ctx.session._retval = ev.result or None
+        ctx.session.write_stream_event(None)  # FIXME: send a special event instead?
```

Comment:
Is it too big to include this special event in this PR?

Reply:
Resolved!